CRM/Core/Reference/Basic.php

```diff
@@ -113,7 +113,7 @@ public function findReferences($targetDao) {
   public function getReferenceCount($targetDao) {
     $targetColumn = $this->getTargetKey();
     $params = [
-      1 => [$targetDao->$targetColumn, 'String'],
+      1 => [$targetDao->$targetColumn ?? '', 'String'],
```

Comment:
@seamuslee001 I've updated this to bypass the query if null rather than 'make it work' after more thought/discussion. I did adopt your syntax although I think they are probably equally ok